sindresorhus/refined-github
# Restore `tags-dropdown` link to open tag pages

Closes #2658 

# Test
1. Open any repo's releases page. (e.g. https://github.com/sindresorhus/refined-github/releases)
2. Open the "Select tag" dropdown and click on any tag. (e.g. Selecting 19.12.22)
3. Observer the behavior, it should redirect to the tag page. (https://github.com/sindresorhus/refined-github/releases/tag/19.12.22)
